workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Config/LocatorConfigTests.cs

```diff
@@ -43,26 +43,47 @@ public void Validate_should_return_false_when_LocatorHost_is_empty()
         public void Validate_should_return_false_when_ProjectName_is_empty()
         {
             var config = GetDefaultWorkingConfig();
-            config.LocatorParameters.CredentialsType = LocatorCredentialsType.LoginToken;
-            config.LocatorParameters.LoginToken.Token = "some token who cares";
             config.LocatorParameters.ProjectName = "";
 
             var exception = Assert.Throws<ConnectionFailedException>(() => config.Validate());
             Assert.IsTrue(exception.Message.Contains("projectName"));
         }
 
         [Test]
-        public void Validate_should_return_false_when_LoginToken_is_empty()
+        public void Validate_should_return_false_when_LoginToken_is_empty_for_login_token_flow()
         {
             var config = GetDefaultWorkingConfig();
             config.LocatorParameters.CredentialsType = LocatorCredentialsType.LoginToken;
             config.LocatorParameters.LoginToken.Token = "";
-            config.LocatorParameters.ProjectName = "some project who cares";
 
             var exception = Assert.Throws<ConnectionFailedException>(() => config.Validate());
             Assert.IsTrue(exception.Message.Contains("loginToken"));
         }
 
+        [Test]
+        public void Validate_should_return_false_when_DeploymentTag_empty_for_steam_flow()
+        {
+            var config = GetDefaultWorkingConfig();
+            config.LocatorParameters.CredentialsType = LocatorCredentialsType.Steam;
+            config.LocatorParameters.Steam.DeploymentTag = "";
+            config.LocatorParameters.Steam.Ticket = "something";
+
+            var exception = Assert.Throws<ConnectionFailedException>(() => config.Validate());
+            Assert.IsTrue(exception.Message.Contains("deploymentTag"));
```

Comment:
I thought we said we don't want to check for keywords anymore?

Reply:
Removed the keyword checks.